xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleTagSami.cpp

```diff
@@ -14,6 +14,56 @@
 #include "utils/RegExp.h"
 #include "utils/StringUtils.h"
 
+#include <algorithm>
+#include <map>
+
+namespace
+{
+// W3C HTML 16 basic colors
+const std::map<std::string, std::string> HTMLCOLORS = {
```

Comment:
I think this makes more sense in the utils::color namespace. Also we have a constexpr map in Kodi added by Lukas a while ago. E.g. search the codebase by `constexpr auto formatMap`

Reply:
i had placed here so that to remove them with future rework
but no problem i move on utils by using Color type
i was not aware of constexpr i will check thanks

Reply:
changes done